app/controllers/accounts_controller.rb

```diff
@@ -24,6 +24,11 @@ def show
       format.rss do
         expires_in 1.minute, public: true
 
+        if @account&.user&.setting_norss == true
```

Comment:
```suggestion
        if @account&.user&.setting_norss
```
Boolean values don't have to be explicitly checked for '== true' or '== false'